### ISSUE TITLE
automate the conversion of parameters when `time.unit != 7`

### ISSUE THREAD

**Issue by AdrienLeGuillou** [EpiModel/EpiModel#815]
questions:

- an outside function to run after `param.net`?
- a function inside `netsim`?
- simple `time.unit` conversion? or also rate2int, int2rate, etc?

@smjenness

**Comment by smjenness** [comment]
I am thinking the first option. First you run the parameters through param.net or param_msm with parameters defined based on the default time.unit (=7), and then this param_timeunit_convert function would take (param, time.unit) and apply the conversion automatically to all parameters ending in .rate and .int.

**Comment by AdrienLeGuillou** [comment]
I agree, it will avoid involuntary conversion

**Comment by AdrienLeGuillou** [comment]
Agreed, and even for EpiModelHIV, we would also need a set of convention to know which parameter require conversion and which do not
